src/client/connect.rs

```diff
@@ -110,7 +112,8 @@ where
 impl<A, B, C> Future for ConnectFuture<A, B, C>
 where
     C: MakeConnection<A>,
-    B: Payload,
+    B: HttpBody,
+    LiftBody<B>: hyper::body::Payload,
```

Comment:
and this

Reply:
You really wont let me have my shorthand bounds, will you D: